tailwind.base.css

```diff
@@ -4,7 +4,7 @@
 
 @layer components {
   .card-shadow {
-    @apply shadow-xl hover:shadow-primary-lightest transition-shadow;
+    @apply shadow-xl hover:shadow-xl-hover transition-shadow;
```

Comment:
Not directly related, but maybe we should also adapt the shadow size and use `hover:shadow-xl` instead of `hover:shadow-2xl` in the `OrganisationPreviewComponent` to be consistent. Not sure about the color, though. I think the final UX recommendation was a border in this case anyway.

Reply:
There is a story to rework the organisation grid.